CMakeLists.txt

```diff
@@ -38,9 +38,9 @@ if ("${CMAKE_SYSTEM_NAME}" MATCHES "Darwin")
 endif()
 
 # Set RPATH if not installing to a system library directory
-list(FIND CMAKE_PLATFORM_IMPLICIT_LINK_DIRECTORIES "${CMAKE_INSTALL_PREFIX}/lib" INSTALL_IS_SYSTEM_DIR)
+list(FIND CMAKE_PLATFORM_IMPLICIT_LINK_DIRECTORIES "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}" INSTALL_IS_SYSTEM_DIR)
```

Comment:
This pattern is used in several places. Here, in https://github.com/puppetlabs/cpp-pcp-client/pull/234/files, and https://github.com/puppetlabs/facter/pull/1698/files. I wonder if it would be useful to make it a macro in leatherman, something like `leatherman_set_cmake_install_rpath`?

Reply:
Yeah, would be useful.